layouts/community/ergodox/meagerfindings/keymap.c

```diff
@@ -0,0 +1,566 @@
+    * |        |      |      |Cp/Cut|paste |      |------|           |------|   +  |   4  |   5  |   6  |   +  |        |
+    * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+    * |        |      |      |      |      |      |      |           |      |   -  |   1  |   2  |   3  |   /  |        |
+    * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
+    *   |      |      |      |      |      |                                       |   0  |   .  |  00  |  00  | Enter|
+    *   `----------------------------------'                                       `----------------------------------'
+    *                                        ,-------------.       ,-------------.
+    *                                        |      |      |       |      |      |
+    *                                 ,------|------|------|       |------+------+------.
+    *                                 |      |      |      |       |      |Nupmad|      |
+    *                                 |      |      |------|       |------|Enter |      |
+    *                                 |      |      |      |       |      |      |      |
+    *                                 `--------------------'       `--------------------'
+    */
+    // Numpad  Layer
+    [NUM] = LAYOUT_ergodox(
+      //left hand
+      _______,        _______,     _______,     _______,        _______,        _______,    _______,
+      _______,        _______,     _______,     _______,        _______,        _______,    _______,
+      _______,        _______,     _______,   TD(TD_COPY_CUT),  PAST_PS,        _______,
+      _______,        _______,     _______,     _______,        _______,        _______,    _______,
+                      _______,     _______,     _______,        _______,        _______,
+                                                                                            _______,    _______,
+                                                                                                        _______,
+                                                                            _______,        _______,    _______,
+
+      //right hand
+      _______,      _______,      _______,       _______,     _______,     _______,        _______,
+      _______,     KC_EQUAL,      KC_7,          KC_8,        KC_9,        KC_ASTR,        _______,
+                    KC_PLUS,      KC_4,          KC_5,        KC_6,        KC_PLUS,        _______,
+      _______,     KC_MINUS,      KC_1,          KC_2,        KC_3,        KC_KP_SLASH,    _______,
+                    KC_KP_0,      KC_DOT,        DBLE_ZER0,   DBLE_ZER0,   KC_KP_ENTER,
+      _______,      _______,
+      _______,
+      _______,      KC_PENT,      _______),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
This function is deprecated and should be removed.

Your keymap doesn't use this functionality at all.

Reply:
Removed